includes/orphaned_objects.inc

```diff
@@ -0,0 +1,252 @@
+<?php
+
+/**
+ * @file
+ * A list of orphaned Islandora objects.
+ */
+
+/**
+ * Builds the management form for the Orphaned Objects module.
+ *
+ * @param array $form
+ *   An array representing a form within Drupal.
+ * @param array $form_state
+ *   An array containing the Drupal form state.
+ *
+ * @return array
+ *   An array containing the form to be rendered.
+ */
+function islandora_orphaned_objects_manage_form($form, $form_state) {
+  if (isset($form_state['show_confirm'])) {
+    $form['confirm_message'] = array(
+      '#type' => 'item',
+      '#markup' => format_plural(count($form_state['pids_to_delete']),
+      'Are you sure you want to delete the 1 object?',
```

Comment:
The irreversible nature of the action should be mentioned in these confirmations. In the case where there is a single object to be purged it should probably be present with a link. This could be extended to whenever the number (10 and under?) of objects is manageable UX wise.

Reply:
Adding with next commit.

Reply:
The way I've implemented this lists the PIDs if there are 10 or fewer. Any reason this shouldn't be expanded? What's a reasonable limit... 30? 50?